libethash-cuda/ethash_cuda_miner.cpp

```diff
@@ -69,7 +67,11 @@ ethash_cuda_miner::ethash_cuda_miner()
 	m_light = new hash64_t*[devicesCount];
 	for(int i = 0; i < devicesCount; i++)
 		m_light[i] = nullptr;
-	m_dag = nullptr;
+}
+
+ethash_cuda_miner::~ethash_cuda_miner()
+{
+	delete[] m_light;
```

Comment:
This will not be needed at at all if `std::vector` used.

Reply:
Done